src/modules/colorPicker/ColorPickerUI/App.xaml.cs

```diff
@@ -27,23 +27,27 @@ protected override void OnStartup(StartupEventArgs e)
             _args = e?.Args;
 
             // allow only one instance of color picker
-            _instanceMutex = new Mutex(true, @"Global\ColorPicker", out bool createdNew);
+            _instanceMutex = new Mutex(true, @"Local\ColorPicker", out bool createdNew);
```

Comment:
Why do we need to change the name of the mutex?

Reply:
I think we should have more general naming for kernel objects, anyway. Something like PowerToys_ModuleName_Guid.

Reply:
I agree with the proposal, but the mutex itself should be local. We can unify the names of mutexes when we tackle #10541.

Reply:
Using `PowerToys_ModuleName` for the mutex name is a good idea, we don't need the GUID since we may want to use `PowerToys_ModuleName_something` for different events for the same module.

Reply:
I changed the name of the mutex, marking this as resolved.